bin/BuildPackages.sh

```diff
@@ -29,6 +29,7 @@ COLORS=yes
 STRICT=no       # exit with non-zero exit code when encountering any failures
 PARALLEL=no
 PACKAGES=()
+NCORE=$(sudo dmidecode -t 4 | grep 'Core Enabled:' | awk '{a+=$NF}END{print a}')
```

Comment:
This is not portable (e.g. no `dmidecode` on macOS), and requires root access, both of  which are an absolute blocker, I am afraid.

A better way would be to offer a command line option to override the number of parallel tasks to use.